src/client/components/ErrorModal.tsx

```diff
@@ -66,16 +66,40 @@ const ErrorModalWrapper = styled(Modal)`
     max-width: min(36rem, 100% - 2rem);
     min-height: auto;
   }
-  ${Button} {
```

Comment:
These style the close button, but if that button is used it's rendered in ModalNew and should be styled there